src/components/views/rooms/EventTile.tsx

```diff
@@ -1210,6 +1210,7 @@ export class UnwrappedEventTile extends React.Component<EventTileProps, IState>
                         "ref": this.ref,
                         "className": classes,
                         "aria-live": ariaLive,
+                        "aria-label": this.props.mxEvent.sender.name + '. ' + (this.props.mxEvent.clearEvent || this.props.mxEvent.event).content.body,
```

Comment:
I'm not sure this will work, the `content.body` isn't always what should be read out, as it'll include fallbacks for replies, threads etc, and be just a filename for media events. Maybe an `aria-describedby` pointing at an ID scheme like `mx_EventTile_${eventId}` or similar

Reply:
Thanks for the suggestion, I've now updated the PR to use `aria-labelledby`.